variables.tf

```diff
@@ -14,104 +14,105 @@
  * limitations under the License.
  */
 
-variable project {
+variable "project" {
   description = "The project to deploy to, if not set the default provider project is used."
   default     = ""
 }
 
-variable region {
+variable "region" {
   description = "Region for cloud resources"
   default     = "us-central1"
 }
 
-variable ip_version {
+variable "ip_version" {
   description = "IP version for the Global address (IPv4 or v6) - Empty defaults to IPV4"
   default     = ""
 }
 
-variable firewall_networks {
+variable "firewall_networks" {
   description = "Names of the networks to create firewall rules in"
-  type        = "list"
+  type        = list(string)
   default     = ["default"]
 }
 
-variable firewall_projects {
+variable "firewall_projects" {
   description = "Names of the projects to create firewall rules in"
-  type        = "list"
+  type        = list(string)
   default     = ["default"]
 }
 
-variable name {
+variable "name" {
   description = "Name for the forwarding rule and prefix for supporting resources"
 }
 
-variable target_tags {
+variable "target_tags" {
   description = "List of target tags for health check firewall rule."
-  type        = "list"
+  type        = list(string)
 }
 
-variable backends {
+variable "backends" {
   description = "Map backend indices to list of backend maps."
-  type        = "map"
+  type        = map
```

Comment:
IMO, this should become a list as backend_params is a list, but this is not what this PR is about :)